sunpy/instr/aia/aia.py

```diff
@@ -146,30 +151,27 @@ def aia_inst_genx_to_dict(self, path_to_file):
                     key = wavelength
 
                     # store information from those files in dataframe
-                    # TODO: try to refactor this to get rid of this many four loops
+                    # TODO: refactor to get rid of so many four loops
                     for value in data[name]:
                         for prop in self.properties:
                             wave_dictionary[prop] = [value[prop][0]]
+            # store in dictionary
             self.data_dictionary[wavelength] = wave_dictionary
```

Comment:
Probably best to either expose `data_dictionary` __or__ `dataframe` through `self`. No need for both of them if they contain the same information.